weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -18,7 +21,9 @@ export const CallSummary: React.FC<{
   );
   const summary = _.fromPairs(
     Object.entries(span.summary ?? {}).filter(
-      ([k, a]) => !k.startsWith('_') && k !== 'latency_s' && a != null
+      ([k, a]) =>
+        // Display all summary fields, but remove usage stats because we have a separate table
+        !k.startsWith('_') && k !== 'latency_s' && a != null && k !== 'usage'
```

Comment:
can we make a const called `SUMMARY_FIELDS_EXCLUDED_FROM_GENERAL_RENDER` and add `latency_s and `usage`?